doc/rst/mason-packages/guide/manifestfile.rst

```diff
@@ -0,0 +1,76 @@
+:title: Mason Guide: *Manifest*
+
+The Manifest File
+=================
+
+The ``Mason.toml`` manifest file is written in TOML(for more information see TOML section below).
+Each time a new package is created in Mason a standard TOML file is included in the top-level
+directory of the package.
+
+For example, ``Mason.toml``:
+
+.. code-block:: text
+
+    [brick]
+    authors = ["Sam Partee <Sam@Partee.com>"]
+    chplVersion = "1.16.0"
+    license = "None"
+    name = "MyPackage"
+    version = "0.1.0"
+
+    [dependencies]
+    curl = '1.0.0'
+
+The ``chplVersion`` field indicates Chapel releases compatible with this
+package. There are a number of accepted formats:
+
+.. code-block:: text
+
+    "1.16.0"         # 1.16.0 or later
+    "1.16"           # 1.16.0 or later
+    "1.16.0..1.19.0" # 1.16 through 1.19, inclusive
+
+By default, ``chplVersion`` is set to represent the current Chapel release or
+later. For example, if you are using the 1.16 release, chplVersion will be
+``1.16.0``.
+
+The ``license`` field indicates the software license under which the package is distributed.
+Any of the licenses available at the `SPDX License List <https://spdx.org/licenses/>`_ can be used for Mason packages.
+The license field defaults to ``None``.
+
+TOML is the configuration language chosen by the chapel team for
+configuring programs written in chapel. A TOML file contains the
+necessary information to build a chapel program using mason.
+`TOML Spec <https://github.com/toml-lang/toml>`_.
```

Comment:
This link displays a little oddly in the documentation.  Maybe a better lead up to it?